src/hermezjs-typings.d.ts

```diff
@@ -136,12 +140,47 @@ declare module "@hermeznetwork/*" {
       | "TransferToBJJ"
       | "TransferToEthAddr"
       | "Withdrawn";
+    // batchNum: number | null;
+    // info: string | null;
+    // maxNumBatch: number;
+    // requestAmount?: unknown;
+    // requestFee?: unknown;
+    // requestFromAccountIndex?: unknown;
+    // requestNonce?: unknown;
+    // requestToAccountIndex?: unknown;
+    // requestToBJJ?: unknown;
+    // requestToHezEthereumAddress?: unknown;
+    // requestTokenId?: unknown;
+  };
+
+  // ToDo: According to the docs: https://apidoc.hermez.network/#model-TransactionType supported types are:
```

Comment:
What is the ToDo? What is missing?

Reply:
We miss resolving the inconsistency between the types declared in the docs and the types present in the code:

![image](https://user-images.githubusercontent.com/15896177/134402292-499f0669-d002-45bd-9929-8cd666a56b7c.png)

Reply:
In general, the protocol has way more features than the ones we need. We also add a type just for simplicity internally. I think it's good as is. I don't expect us to support the missing types. Definitely not in the mid-term

Reply:
When you refer to "the ones *we* need" who do you mean by "we"? "We" as the creators of the `wallet-ui` frontend or "we" as the creators of the `hermezjs` library? I ask because the module `hermezjs-typings.d.ts` should implement any feature available in the library. In other words:

 * Are we ok not supporting in the library the types `CreateAccountDepositTransfer`, `DepositTransfer` and `ForceTransfer` present in the docs  because any user of the lib would need these types implemented by the protocol?
* Is the type `Withdrawn` we are adding generic enough to be used by any client of the library? Or is it just something required by our `wallet-ui` that would not make sense to other users of the lib?

Reply:
So Withdraw is the L1 part of the withdrawal process, so it is general enough, and hermezjs has a utility that takes care of the Withdraw.

And we are okay not supporting the other Transfer types, at least for now. If the use case comes up, we will add it. To prioritize features, we only add them when someone (or us) requests them

Reply:
Ok! Comment removed. Thx for the clarification.